geo-location.html

```diff
@@ -134,6 +149,19 @@
      *   @param {Number} detail.longitude Longitude of the current position.
      */
 
+    attached: function() {
+      if ('permissions' in navigator) {
+        navigator.permissions.query({
+          name: 'geolocation'
+        }).then(function(permissionStatus) {
+          this._setPermissionStatus(permissionStatus.state);
+          permissionStatus.onchange = function() {
```

Comment:
What happens if this element gets attached/detached/attached from the DOM? Does `permissionStatus` get gc'd properly and we're not adding creating a bunch of orphaned change handlers?

Reply:
@ebidel I dont know. Could you please help me do this in a better way?

Reply:
As the author of this PR, I was hoping you could do that research.

Reply:
@ebidel BTW, what about placing this code inside `ready` lifecycle callback instead of `attached`? Is it a bad idea?
 `attached` can be called multiple times during the lifetime of an element. The first `attached` callback is guaranteed not to fire until after `ready`.